### ISSUE TITLE
Logout is asking me to login

### ISSUE THREAD

**Issue by johnpapa** [Azure/static-web-apps#344]
I am logged in as github (was there for days, so that must remember and not time out).

I click the logout link, and it used to log me out. Now it asks me to log in (see the video below)

https://user-images.githubusercontent.com/1202528/113364159-c9def200-9320-11eb-8912-eb0c0f0f930e.mp4

**Comment by johnpapa** [comment]
update: I clicked the first option for logging out (logging into MSFT email) and it returned me back to the app, still logged in with github. then I tried logout again, this time I choose other options, which them threw an error 401. I went back to shopathome.dev and was logged in still. So then I logged out again, clicked the MSFT option, and it returned me to the app logged out.

I have no idea what is happening. lol

but now I am logged out. 

i've had this site for a while ... do I have perhaps some old setting?
